components/Modal/index.css

```
@@ -0,0 +1 @@
+@import "./Modal.module.css";
```

```suggestion
@import "./Modal.module.css";
@import "./sub-components/Content.module.css";
@import "./sub-components/Header.module.css";
@import "./sub-components/Title.module.css";
```
Storybook is correctly rendering the styles of the component and its sub-components, but without the imports above, our css build tool will not package correctly. It's a bit confusing, we plan to improve this in the future.

---

Done!